src/healthchecks_io/schemas/checks.py

```diff
@@ -14,34 +14,36 @@
 
 import pytz
 from croniter import croniter
-from pydantic import BaseModel
+from pydantic import field_validator, BaseModel
 from pydantic import Field
 from pydantic import validator
 
 
 class Check(BaseModel):
     """Schema for a check object, either from a readonly api request or a rw api request."""
 
-    unique_key: Optional[str]
+    unique_key: Optional[str] = None
     name: str
     slug: str
-    tags: Optional[str]
-    desc: Optional[str]
+    tags: Optional[str] = None
+    desc: Optional[str] = None
     grace: int
     n_pings: int
     status: str
-    last_ping: Optional[datetime]
-    next_ping: Optional[datetime]
+    last_ping: Optional[datetime] = None
+    next_ping: Optional[datetime] = None
     manual_resume: bool
-    methods: Optional[str]
+    methods: Optional[str] = None
     # healthchecks.io's api doesn't return a scheme so we cant use Pydantic AnyUrl here
-    ping_url: Optional[str]
-    update_url: Optional[str]
-    pause_url: Optional[str]
-    channels: Optional[str]
-    timeout: Optional[int]
-    uuid: Optional[str]
-
+    ping_url: Optional[str] = None
+    update_url: Optional[str] = None
+    pause_url: Optional[str] = None
+    channels: Optional[str] = None
+    timeout: Optional[int] = None
+    uuid: Optional[str] = None
+
+    # TODO[pydantic]: We couldn't refactor the `validator`, please replace it by `field_validator` manually.
+    # Check https://docs.pydantic.dev/dev-v2/migration/#changes-to-validators for more information.
     @validator("uuid", always=True)
     def validate_uuid(cls, value: Optional[str], values: Dict[str, Any]) -> Optional[str]:  # noqa: B902
```

Comment:
Validator here that needs to be converted to a field_validator

Reply:
Good catch! I've updated this and removed the TODO.